ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -473,7 +473,7 @@ export default function ViewQuote() {
             onClose={() => setSelectQuotePopoverShown(false)}
             onSubmit={(aggId) => {
               dispatch(setSelectedQuoteAggId(aggId))
-              dispatch(setCustomGasLimit(null))
+              dispatch(swapCustomGasModalLimitEdited(null))
```

Comment:
Ensures the custom gas limit is reset if the user selects a different quote.

Reply:
I think so, sort of... I think the correct improvement to make here is what I have now done in 4444bac82, which might be what you are talking about

Reply:
Yep, that's good!